test/property.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const test = require('tape')
+const fc = require('fast-check')
+const {isObject, isPlainObject, pick, cloneDeep} = require('lodash')
+const is = require('..')
+
+test('unist-util-is properties', (t) => {
+  t.plan(4)
+  t.doesNotThrow(
+    () =>
+      fc.assert(
+        fc.property(fc.record({type: fc.string({minLength: 1})}), (node) =>
+          is(node)
+        )
+      ),
+    'should check if given node (#1)'
+  )
+
+  t.doesNotThrow(
+    () =>
+      fc.assert(
+        fc.property(
+          fc
+            .dictionary(fc.string(), fc.string())
+            .filter((node) => typeof node.type === 'undefined'),
+          (node) => !is(node)
+        )
+      ),
+    'should check if given node (#2)'
```

Comment:
```suggestion
    'should see any object w/o a `type` as a non-node'
```

Reply:
would it make sense to update https://github.com/syntax-tree/unist-util-is/blob/18bbf9081ade2cc26eb7f3958c5cbdfbd3d9fc91/test.js#L76  as well?